src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,14 @@
+{
+  "schemaVersion": 1,
+  "id": "universalcraft",
+  "name": "UniversalCraft",
+  "version": "1.0.0",
+  "environment": "*",
+  "authors": [
+    "Sk1erLLC"
+  ],
+  "depends": {
+    "fabricloader": "*"
+  },
+  "description": "UniversalCraft"
+}
```

Comment:
Explanation:
Every fabric mod and library requires a ``fabric.mod.json`` to be loaded on the same classloader as the other mods, this allows UC to access Minecraft classes correctly